### PR TITLE
Feature/main navigantion component

### PR DESCRIPTION
**Pull request by Satira**
Main Navigation Component consist of following features:

- Sticky main navigation bar
- Right-hand side menu overlay
- Left-hand side menu overlay

### CODE REVIEW

File: source/js/main-navigation.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+$(document).ready(function() {
+  $('.icon').click(function() {
+    $('.icon').toggleClass('active');
+  })
+});
+
+//Main Menu Overlay
+$(document).ready(function() {
+  $('.icon').click(function() {
+    $('.right-menu-overlay').toggleClass('on') && $('.left-menu-overlay').toggleClass('on');
```
Comment (chemicalbr) [inline source/js/main-navigation.js]: this seems wrong
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_Operators

File: source/style/base/_vars.scss [hunk 2]
```diff
@@ -3,23 +3,22 @@
...
+$color-font-h: hsla(345, 81%, 64%, 1);
+$color-bg-hf: hsla(0, 0%,0%, 1.0);
+$color-bg: hsla(260, 12%, 20%, 1.0);
 
 //Breakpoints map
 $breakpoints: (
-  'sm': 1279px,
-  'lg': 1280px,
+  'sm': 767px,
+  'lg': 768px,
```
Comment (chemicalbr) [inline source/style/base/_vars.scss]: seems like you need just one not two breakpoints
;)

File: source/style/modules/preloader.scss [hunk 3]
```diff
@@ -0,0 +1,100 @@
+#loader-wrapper {
```
Comment (chemicalbr) [inline source/style/modules/preloader.scss]: do not use id for styling

File: source/style/modules/sticky-nav.scss [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+
+
+//Left panel content
+
+@include mq(sm){
+  .left-menu-overlay{
+    display: none;
+  }
+}
+@include mq(lg){
```
Comment (chemicalbr) [inline source/style/modules/sticky-nav.scss]: space between ){

File: source/index.html [hunk 5]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" xmlns="http://www.w3.org/1999/html">
```
[the remark this answers is not in the document]
Comment (Satira) [reply source/index.html]: Agree. 
However I don't know how this line had been added.
Anyway. It's fixed in 06de6c69c95e356f51227fb443da0215aae3ee2e

File: source/index.html [hunk 6]
```diff
@@ -15,9 +15,86 @@
   <meta name="keywords" content="JS, HTML, CSS3, Flexbox">
   <meta name="author" content="Nikifor Nikolov">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="/build/apollo.css?1d2fe3436737de1d6c75" rel="stylesheet">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.5.0/css/all.css" integrity="sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU" crossorigin="anonymous">
 </head>
 
 <body>
+<!--Loader -->
```
[the remark this answers is not in the document]
Comment (Satira) [reply source/index.html]: Hmm, It has already done in 60c8a099b5999944cace90a4cc3b77eb0507aa43

File: source/js/main-navigation.js (hunk 1 above)
Comment (Satira) [reply source/js/main-navigation.js]: You are right. I don't need logical operator in this case.
Fixed in 06de6c69c95e356f51227fb443da0215aae3ee2e

File: source/style/base/_vars.scss (hunk 2 above)
Comment (Satira) [reply source/style/base/_vars.scss]: :) :)
well, you are right. It is just one bp at the moment.
But let me keep this for now as I want to add a few more in near future (and will extend breakpoints map)

File: source/style/modules/preloader.scss (hunk 3 above)
Comment (Satira) [reply source/style/modules/preloader.scss]: hmm this one has been fixed in 672290b7081b9a192a3d6af6aa8a013fe1ad856b

File: source/style/modules/sticky-nav.scss (hunk 4 above)
Comment (Satira) [reply source/style/modules/sticky-nav.scss]: Resolved in 06de6c69c95e356f51227fb443da0215aae3ee2e

File: source/style/base/_vars.scss (hunk 2 above)
Comment (chemicalbr) [reply source/style/base/_vars.scss]: ok ;]]

**Pull request merged**